contracts/SnappAuction.sol

```diff
@@ -170,7 +175,10 @@ contract SnappAuction is SnappBase {
         );
         auctions[auctionIndex].shaHash = nextAuctionHash;
 
-        emit SellOrder(auctionIndex, auctions[auctionIndex].size, accountId, buyToken, sellToken, buyAmount, sellAmount);
+        emit SellOrder(
+            auctionIndex, auctions[auctionIndex].size, accountId,
+            abi.encode((uint(buyToken) << 16) + (uint(sellToken) << 24) + (uint(sellAmount) << 32) + (uint(buyAmount) << 128))
```

Comment:
I don't think this event should change, but if let's please reuse the `encodeOrder` function.

Reply:
The code will be killed anyways(including the event) with your PR, right?

Reply:
I will rebase it on your PR.